packages/cli/src/ui/components/shared/text-buffer.test.ts

```diff
@@ -1736,12 +1755,22 @@ describe('textBufferReducer vim operations', () => {
         cursorRow: 1,
         cursorCol: 0,
         preferredCol: null,
-        visualLines: [['line1'], ['line2'], ['line3'], ['line4']],
-        visualScrollRow: 0,
-        visualCursor: { row: 1, col: 0 },
-        viewport: { width: 10, height: 5 },
+        visualLayout: {
```

Comment:
we should refactor these tests to not require this. specifying the visualLayout and all this in the test has lost the thread. These should get computed from the initial state  or be empty rather than get filled in.
I care because otherwise these tests are overly verbose and painful to maintain.